docs/usage/trusted-tls-for-garden-runtime.md

```diff
@@ -0,0 +1,43 @@
+# Trusted TLS Certificate for Garden Runtime Cluster
+
+In Garden Runtime Cluster components are exposed via `Ingress` resources, which make them addressable under the HTTPS protocol.
+
+Examples:
+- Plutono
+
+Gardener generates the backing TLS certificates, which are signed by the shoot cluster's CA by default (self-signed).
+
+Unlike with a self-contained Kubeconfig file, common internet browsers or operating systems don't trust a garden runtime's cluster CA and adding it as a trusted root is often undesired in enterprise environments.
+
+Therefore, Gardener operators can predefine trusted wildcard certificates under which the mentioned endpoints will be served instead.
+
+## Register a trusted wildcard certificate
+Garden Runtime Cluster components are published under the ingress domain (`operator.gardener.cloud/v1alpha1.Garden.spec.runtimeCluster.ingress.domain`) a wildcard certificate is required.
```

Comment:
```suggestion
Since garden runtime cluster components are published under the ingress domain (`operator.gardener.cloud/v1alpha1.Garden.spec.runtimeCluster.ingress.domain`) a wildcard certificate is required.
```